jni_gen_test/java/dev/dart/jni_gen_test/Example.java

```diff
@@ -0,0 +1,53 @@
+package dev.dart.jni_gen_test;
+
+public class Example {
+	// static final fields
+	public static final int ON = 1;
+	public static final int OFF = 0;
+
+	// static fields
+	public static Aux aux;
+	public static int num;
+
+	static {
+		aux = new Aux(true);
+		num = 121;
+	}
+
+	// static methods
+	public static Aux getAux() {
+		return aux;
+	}
+
+	public static int addInts(int a, int b) {
+		return a+b;
```

Comment:
Is this default Java formatting? No space between the + and its operands? Please run a formatter.

(Does Java really use tabs by default to format. 😭 Can we change it to spaces? 💪 )

Reply:
I should have run google-java-format on it (As rest of the summarizer).

Reply:
Maybe run the formatter on CI as well, that way we don't break it again later. 🥇

Reply:
It will require to either download the jar from google-java-format release or use a third party GH action (axel-op/googlejavaformat-action).

Which one is preferred?

Reply:
The easiest one. (Which is the GitHub action I would think, because it downloads the jar for us.)